src/server/editorServices.ts

```diff
@@ -2051,7 +2067,6 @@ namespace ts.server {
                 this,
                 this.documentRegistry,
                 cachedDirectoryStructureHost);
-            // TODO: We probably should also watch the configFiles that are extended

```

Comment:
As far as I can tell, this was actually a red herring because the project's `CompilerOptions` (most importantly `configFile`) have not been calculated at this point. Not sure if there are other comments like this that should be removed with this PR. 😄